### DISCUSSION TITLE
Getting username and email of user

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3307] [Q&A]
Hi, 
I would like to know what are the variables to be used for username and email id for QFIELD . 
Please help me in getting list of all Qfield variable.

I am aware of how to get cloud username and email id. Also share me other variables that can be helpful in qfield Cloud

**Comment by suricactus** [comment]
All QGIS variables are available in QField forms. Also see https://docs.qfield.org/how-to/variables/ to setup custom forms. 

Also, please be more specific what kind of variables you need. Thanks.
